### ISSUE TITLE
Password reset form redirects to login.

### ISSUE THREAD

**Issue by robbykrlos** [enso-ui/auth#3]
This is  a **question**.

### Prerequisites
* [4.5.0] Are you running the latest version?
* [x] Are you reporting to the correct repository?
(enso is made of many specialized packages: https://github.com/laravel-enso)
* [x] Did you check the documentation?
* [x] Did you perform a cursory search?

### Description
Hello,

I have a question, hope a stupid one, but I give up after searching both BE and FE for some answers and nothing seems to scream 'bug'.

As the title mentions, when a password reset is triggered, and the email is received, the link will redirect to ``http://mylocalhost/password/reset/2F869e2743efd8de73b0b5f569663369a0a2b6e4e54279998ce99d997c8da9ce89`` which should normally show the reset form (``@enso-ui/auth/src/bulma/pages/auth/password/Reset.vue``). What happens on our side is - the page is loaded, the title shows a bit, the form sometimes shows to for a fraction of a second, but then the login is loaded (url : http://mylocalhost/login). Most of the times the load is so fast that you basically see the login page after you click the reset link.

This generates no ajax calls (XHR). We've have checked BE :
- laravel-enso/core/routes/app/auth.php
- laravel-enso/core/src/Http/Controllers/Auth/*.php
- vendor/laravel/ui/auth-backend/ResetsPasswords.php

At some point we figured that if you go back (using browser back function) it will load the reset form and the functionality works just fine - even more our debugging code was triggered in BE when Post routes are fired on submit. 

All of these things made us believe it must be something in FE. Checked:
- @enso-ui/auth/src/bulma/pages/auth/password/Reset.vue 
- @enso-ui/auth/src/bulma/pages/auth/password/Email.vue
- @enso-ui/auth/src/bulma/pages/auth/components/AuthForm.vue
- @enso-ui/auth/src/bulma/pages/auth/Login.vue
- @enso-ui/auth/src/bulma/routes/auth.js

But found nothing that seemed wrong. I even tried (paranoia) to comment some setTimeouts just to be sure that some anomaly isn't triggering the redirect to login for some reason on my env.

We've tested on 3 different webservers (thinking that apache .htaccess might have some impact on URI rewrites). no difference.
We've tried 2 browsers.
We've double checked .env configurations - thinking it can be something about the token validation.

Any chance you have seen this before and have some quick fix?
Thanks

**Comment by aocneanu** [comment]
@raftx24 I believe you did something recently for a project?

**Comment by raftx24** [comment]
@robbykrlos Hi, thanks for reporting.
it was fixed in the new version 4.6.0

**Comment by robbykrlos** [comment]
That's awesome. Thanks. I'll check the logs - I'm very curious what was it...

**Comment by robbykrlos** [comment]
Hi,

So, we've just upgraded to 4.6.0. The problem is still there. Any other ideas?

PS: I found it hard tot track the 4.6.0 release notes for the fix on this _password reset redirect to login_ issue - I searched several packages from BE and FE and some things but nothing that relevant. We even tested some before we updated to 4.6.0 and had no luck.

**Comment by robbykrlos** [comment]
Hi,

Additional info that may be relevant: we are not using the quotes page:

```php
'showQuote' => false,
```

**Comment by raftx24** [comment]
HI @robbykrlos.
there was an issue [here](https://github.com/enso-ui/ui/commit/39e0ae536198cc36fdd343eb35df292348b337f0) and it was fixed, right now enso 4.6 doesn't have a problem.
Please make sure that you have this  [commit](https://github.com/enso-ui/ui/commit/39e0ae536198cc36fdd343eb35df292348b337f0).

![image](https://user-images.githubusercontent.com/10864136/107193945-ab393a80-6a04-11eb-988f-d976c3f2baba.png)

**Comment by robbykrlos** [comment]
Hi,

We've checked and we do not have that change.

When we updated to 4.6.0, the yarn install came with @enso-ui/ui 4.0.10. Maybe a missed tag? We'll check also package.json to be sure we have the right dependencies.

Generally speaking, shouldn't we follow (keep in sync) with laravel-enso/enso/client/package.json in order to make sure we load the right package dependencies?

Thanks for that, we will make sure we update again, with the right package.
